content/en/docs/contribute/_index.md

```diff
@@ -0,0 +1,622 @@
+---
+title: Contribute
+description: Learn how to contribute to the OpenTelemetry documentation.
+weight: 200
+cSpell:ignore: linktitle open-telemetry prepopulated repos spacewhite
```

Comment:
```suggestion
cSpell:ignore: prepopulated spacewhite
```

> For later: I'll update the cSpell config so that we'll be able to ignore `mermaid` block contents.